modules/@apostrophecms/util/ui/public/video.js

```diff
@@ -1,62 +1,75 @@
-apos.util.widgetPlayers['@apostrophecms/video'] = function(el, data, options) {
+apos.util.widgetPlayers['@apostrophecms/video'] = {
```

Comment:
View this with whitespace changes disabled. Much of it is indented but the same as before.